aqueduct_client/__init__.py

```diff
@@ -3,3 +3,7 @@
 __all__ = [
     'create_client',
 ]
+
+from ._version import get_versions
```

Comment:
Shall we put all the imports at the top? Eventually flake8 will complain about this...

Reply:
yup, will do